src/OSGManipulator/keyboard-manipulator.cpp

```diff
@@ -76,7 +76,7 @@ KeyboardManipulator::KeyboardManipulator(osgViewer::GraphicsWindow *window, int
 
 /// Copy Constructor.
 KeyboardManipulator::KeyboardManipulator( const KeyboardManipulator& fpm, const CopyOp& copyOp )
-   :osg::Object(fpm, copyOp), osg::Callback(fpm, copyOp),inherited( fpm, copyOp )
+   :inherited( fpm, copyOp )
```

Comment:
Do you have any idea why those copy constructors were called this way ?
I saw the same thing is done in `FirstPersonManipulator`, the parent class.

I'm just want to know if there is a specific reason for this. I'm trying to compile using `OpenSceneGraph 3.2` and the class Callback was somehow changed between 3.2 and 3.4. If there was a good reason, this pull-request must be ignore.